src/cli/commands/bitswap/stat.js

```diff
@@ -23,8 +23,7 @@ module.exports = {
         stats.Wantlist = stats.Wantlist || []
         stats.Peers = stats.Peers || []
 
-        console.log(`
-bitswap status
+        console.log(`bitswap status
```

Comment:
As I mentioned in another PR I would rather do that change as a single PR on its own, there are already too many things mixed in here.